.gitignore

```diff
@@ -3,3 +3,4 @@
 /node_modules/
 /spec/**/*.js
 *.dll
+.idea
```

Comment:
This is an editor specific ignore and should be in your global configuration (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore and https://help.github.com/articles/ignoring-files/#explicit-repository-excludes)

Reply:
thx